src/PythERC7412Node.sol

```diff
@@ -26,14 +26,23 @@ contract PythERC7412Node is IExternalNode, IERC7412 {
     function process(
         NodeOutput.Data[] memory,
         bytes memory parameters,
-        bytes32[] memory,
-        bytes32[] memory
+        bytes32[] memory runtimeKeys,
+        bytes32[] memory runtimeValues
     ) external view returns (NodeOutput.Data memory nodeOutput) {
-        (, bytes32 priceId, uint256 stalenessTolerance) = abi.decode(
+        (, bytes32 priceId, uint256 configuredStalenessTolerance) = abi.decode(
             parameters,
             (address, bytes32, uint256)
         );
 
+        uint256 runtimeStalenessTolerance;
+        for (uint256 i = 0; i < runtimeKeys.length; i++) {
+            if (runtimeKeys[i] == "stalenessTolerance") {
+                runtimeStalenessTolerance = uint256(runtimeValues[i]);
```

Comment:
is zero a valid value? if the key is not found, then the value will be zero and hence stalenessTolerance (due to min) will be zero.